chromadb/db/mixins/sysdb.py

```diff
@@ -685,17 +689,28 @@ def _insert_metadata(
                 table.str_value,
                 table.int_value,
                 table.float_value,
+                table.bool_value,
             )
         )
         sql_id = self.uuid_to_db(id)
         for k, v in metadata.items():
-            if isinstance(v, str):
+            if isinstance(v, bool):
```

Comment:
Let's put a comment about why this is first - must come before int since `isinstance(Bool, int)` evaluates to true